### ISSUE TITLE
The current version in pypi (0.4) isn't compatible with ipywidgets 7.x

### ISSUE THREAD

**Issue by ianabc** [takluyver/mobilechelonian#12]
Is it possible to have version 0.5 added to PyPI? Or a note added to the README reminding people  to try installing from the current master branch if they are running ipywidgets 7. f90afaa5f on the  master branch identifies as 0.5 and installs fine with
```
git clone https://github.com/takluyver/mobilechelonian.git
cd mobilechelonian
pip install .
```

PyPI only has [version 0.4](https://pypi.python.org/pypi/mobilechelonian/0.4), which spits out some javascript errors about missing files.

**Comment by hoogenm** [comment]
Thank you for confirming installation compatibility with ipywidgets 7. As an installation command, I believe this will also work: `pip install git+https://github.com/takluyver/mobilechelonian`

 * `pip` and `git` must be installed/available on the system
 * Use an optional `--user` flag if required

**Comment by takluyver** [comment]
Thanks for the reminder - I have released 0.5, so you can now install it from PyPI.
